app/app.test.js

```diff
@@ -279,6 +281,16 @@ describe("app", () => {
                     descending: true,
                   });
                 });
+            })
+            .then(() => {
+              return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then(({ body: { articles } }) => {
+                  expect(articles).toBeSortedBy("created_at", {
+                    descending: true,
+                  });
+                });
```

Comment:
I actually missed this earlier so apologies!
I can see that you're actually chaining tests together in `.then`s here - if they are all in a separate test block then jest will actually run them concurrently. This, if only slightly, will be intentionally slowing down the test suite.
You could always put them in a Promise.all to have them trigger simultaneously, but separate test blocks will achieve the same thing